libs/dbr/context.go

```diff
@@ -47,3 +56,11 @@ func RunsOnRuntime(ctx context.Context) bool {
 	}
 	return v.(bool)
 }
+
+func RuntimeVersion(ctx context.Context) string {
+	if !RunsOnRuntime(ctx) {
+		return ""
+	}
+
+	return ctx.Value(dbrVersionKey).(string)
+}
```

Comment:
Please include a test for this as well, with cases when it is set and when it is not set.